packages/expo-intent-launcher/android/src/main/java/expo/modules/intentlauncher/IntentLauncherModule.java

```diff
@@ -102,6 +102,7 @@ public void startActivity(String activityAction, @NonNull ReadableArguments para
 
     mUIManager.registerActivityEventListener(this);
     mPendingPromise = promise;
+    intent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
```

Comment:
i think this should be possible to set in your JavaScript app code rather than in native code -- could you try setting the `flags` property [here](https://docs.expo.io/versions/v33.0.0/sdk/intent-launcher/#type-intentparams) instead?

Reply:
Thank you for catching it!